src/commands/configure/node.rb

```diff
@@ -18,10 +18,21 @@ def answers_file
           config.node_answers_file(node_name)
         end
 
+        def higher_level_answer_files
+          [
+            config.domain_answers_file,
+            config.group_answers_file(node.primary_group)
+          ]
+        end
+
         private
 
         attr_reader :node_name
 
+        def node
+          Metalware::Node.new(config, node_name)
+        end
+
         def dependency_hash
           dependency_specifications.for_node_in_configured_group(node_name)
```

Comment:
This isn't going to generate the correct `dependency_hash` anymore. Now that that `optional` dependencies are supported, this method skips the checking of `nodes/[node_name].yaml`